website/client/js/services/taskServices.js

```diff
@@ -150,13 +150,6 @@ angular.module('habitrpg')
       });
     };
 
-    function editTask(task, user) {
```

Comment:
Why are you removing this from the task service? The challenges controller also uses this function.